### DISCUSSION TITLE
Wrote a couple plugins to interface with MPD

### DISCUSSION THREAD

**Discussion by thekakkun** [beetbox/beets#4915] [Show and tell]
Hi all,

I wrote some beets plugins to help me manage my music playback in MPD. Check it out!
https://github.com/thekakkun/beets-mpd-utils

- MPD Tracker: Track song plays/skips on MPD.
- MPD DJ: Auto-add songs/albums to your MPD queue.

**Comment by sampsyo** [comment]
These look awesome!

**Comment by arsaboo** [comment]
Can you please add them to the community plugins list... thanks!

**Comment by JOJ0** [comment]
@thekakkun did you ever add them to the community plugins list as @arsaboo suggests? I agree with him that this would be essential! Thanks for the work and really add them please!!! :-)

**Reply by thekakkun** [reply]
Thanks, forgot to update this post, but it was added to the list a while back (https://github.com/beetbox/beets/pull/4917).
